libs/shared/lessons/src/lib/lesson-item/lesson-item.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {
+  Typography,
+  ListItem,
+  Icon,
+  Popover,
+  Button,
+  ButtonGroup,
+  Link,
+} from '@material-ui/core';
+
+import './lesson-item.scss';
+
+export interface Lesson {
+  _id: string;
+  title: string;
+  datime: string;
+  creator?: {
+    _id: string;
+    name: string;
+    imageUrl?: string;
+  };
+  imageUrl: string;
+  description: string;
+}
+
+export interface Action {
+  title: string;
+  action(lesson: Lesson): void;
```

Comment:
Action have action ? it is function or field? if it is function it should look like this: action: (lesson: Lesson) => void